virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -118,6 +118,7 @@ This release adds new features and enhancements related to the following compone
 //CNV-36464: Run a test that shows that the SR-IOV settings are correct
 
 //CNV-34697: Instance types is now GA. Move the 4.14 RN from the TP section to here.
+* Installing and editing xref:../../virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc#virt-creating-vm-instancetype_virt-creating-vms-from-instance-types[customized instance types] and preferences to create a VM from a volume or PersistentVolumeClaim (PVC) was previously Technology Preview and is now generally available.
```

Comment:
I think Web Console is technically a new section, in which case I think VM should be written out because it's the first instance in the section:
```suggestion
* Installing and editing xref:../../virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-instance-types.adoc#virt-creating-vm-instancetype_virt-creating-vms-from-instance-types[customized instance types] and preferences to create a virtual machine (VM) from a volume or PersistentVolumeClaim (PVC) was previously Technology Preview and is now generally available.
```